<!DOCTYPE HTML>
<html lang="en">
<head>
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-140397574-1"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'UA-140397574-1');
  </script>

  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

  <title>Amy X. Lu</title>
  <meta name="author" content="Amy Lu">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" type="text/css" href="stylesheet.css">
  <link rel="icon" type="image/png" href="images/seal.png">
</head>


<!-- Intro -->

<body>
  <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
    <tr style="padding:0px">
      <td style="padding:0px">
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr style="padding:0px">
            <td style="padding:2.5%;width:63%;vertical-align:middle">

              <!-- Title -->
              <p style="text-align:center">
                <name>Amy X. Lu</name>
              </p>

              <!-- Bio -->
              <p>
                I'm a Computer Science PhD student at <a href="https://eecs.berkeley.edu/">UC Berkeley</a> and <a href="https://bair.berkeley.edu/">BAIR</a>, fortunate to be supported by a NSERC PGS-D award. I'm currently spending the summer as a Student Researcher at <a href="https://research.google/teams/brain/">Google Brain</a>.
              </p>
              <p>
                Broadly, I'm interested in developing artificial intelligence methods for the needs of biological data. I'm especially interested in enabling generalization to natural distribution shifts, and biomolecular design for therapeutic uses.
              </p>
              <p>
                Previously, I was at <a href="https://insitro.com">insitro</a>, applying machine learning to accelerate drug discovery. I completed my MSc at the <a href="https://web.cs.toronto.edu/">University of Toronto</a>, during which I was a visitor in Anshul Kundaje's group at <a href="https://sites.google.com/site/anshulkundaje/Home">Stanford University</a>. I was a researcher at <a href="https://www.slizlab.org/">Harvard Medical School</a> prior to my Masters, and completed my undergrad at the <a href="http://doxey.uwaterloo.ca/">University of Waterloo</a>.

              </p>


              <!-- Links -->
              <p style="text-align:center">
                <a href="data/AmyLuCV.pdf">CV</a> &nbsp/&nbsp
                <a href="https://scholar.google.com/citations?user=gQpYbRsAAAAJ">Google Scholar</a> &nbsp/&nbsp
                <a href="https://github.com/amyxlu/">GitHub</a> &nbsp/&nbsp
                <a href="https://twitter.com/amyxlu">Twitter</a> &nbsp/&nbsp
              amyxlu [at] berkeley [dot] edu</a>
            </p>
          </td>

          <!-- Picture -->
          <td style="padding:2.5%;width:40%;max-width:40%">
            <a href="images/vancouver.jpg"><img style="width:100%;max-width:100%" alt="profile photo" src="images/vancouver.jpg" class="hoverZoomLink"></a>
          </td>

        </tr>
      </tbody></table>


      <!-- Publications -->

      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
          <td style="padding:20px;width:100%;vertical-align:middle">
            <heading>Research</heading>
          </td>
        </tr>
      </tbody></table>


      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
          <td style="padding:20px;text-align:left;width:100%;vertical-align:middle">
            <p><em><big>Journal and Conference Papers</big></em></p>
          </td>
        </tr>
      </tbody></table>
      
      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>

        <!-- paper -->
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
            <img src="images/reverse-homology.png" alt="CPCProt" width="160" height="120" style="border-style: none">
          </td>

          <td width="75%" valign="middle">
            <a href="https://www.biorxiv.org/content/biorxiv/early/2021/07/30/2021.07.29.454330.full.pdf" id="idr">
              <papertitle>Discovering molecular features of intrinsically disordered regions by using evolution for contrastive learning</papertitle>
            </a>
            <br>

            Alex X Lu, <strong>Amy X Lu</strong>, Iva Pritišanac, Taraneh Zarin, Julie D Forman-Kay, Alan M Moses
            <br>

            <em>PLOS Computational Biology</em>, 2022
            <br>
            <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010238">Paper</a> / <a href="https://www.biorxiv.org/content/biorxiv/early/2021/07/30/2021.07.29.454330.full.pdf">Preprint</a>
            <p>Reverse Homology is a self-supervised method which captures evolutionary information by contrastive learning to discover molecular features of intrinsically disordered regions.</p>
          </td>
        </tr>


        <!-- paper -->
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
            <img src="images/bioembeddings.png" alt="bio-embeddings" width="160" height="120" style="border-style: none">
          </td>

          <td width="75%" valign="middle">
            <a href="https://currentprotocols.onlinelibrary.wiley.com/doi/pdf/10.1002/cpz1.113" id="bioembeddings">
              <papertitle>Learned embeddings from deep learning to visualize and predict protein sets</papertitle>
            </a>
            <br>
            Christian Dallago, Konstantin Schütze, Michael Heinzinger, Tobias Olenyi, Maria Littmann, <strong>Amy X Lu</strong>, Kevin K Yang, Seonwoo Min, Sungroh Yoon, James T Morton, Burkhard Rost
            <br>

            <venue>Current Protocols</venue>, 2021
            <br>
            <a href="https://currentprotocols.onlinelibrary.wiley.com/doi/pdf/10.1002/cpz1.113">Paper</a> / <a href="https://api.bioembeddings.com/">Web Server</a> / <a href="https://github.com/sacdallago/bio_embeddings">Code</a> 
          </td> 
        </tr>

        <!-- paper -->
        <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/simclr_phylo.png" alt="CPCProt" width="160" height="120" style="border-style: none">
            </td>

            <td width="75%" valign="middle">
              <a href="https://arxiv.org/pdf/2012.13475.pdf" id="evolisallyouneed">
                <papertitle>Evolution Is All You Need: Phylogenetic Augmentation for Contrastive Learning</papertitle>
              </a>
              <br>

              <strong>Amy X Lu</strong>, Alex X. Lu, Alan Moses
              <br>

              <venue>Machine Learning for Computational Biology (MLCB)</venue>, 2020
              <br>
              <a href="https://arxiv.org/pdf/2012.13475.pdf">Paper</a> / <a href="data/EvolIsAllYouNeed_MLCB_Poster.pdf">Poster</a>
              <p>We outline how viewing evolution as natural sequence augmentation for contrastive learning recapitulates comparative genomics, and maximizes the mutual information between sequence and function.</p>
            </td> 
          </tr>

          <!-- paper -->
          <tr>
              <td style="padding:20px;width:25%;vertical-align:middle">
                <img src="images/cpcprot.png" alt="CPCProt" width="160" height="120" style="border-style: none">
              </td>

              <td width="75%" valign="middle">
                <a href="https://www.biorxiv.org/content/10.1101/2020.09.04.283929v2.full.pdf" id="CPCProt">
                  <papertitle>Self-Supervised Contrastive Learning of Protein Representations by Mutual Information Maximization</papertitle>
                </a>
                <br>

                <strong>Amy X Lu</strong>, Haoran Zhang, Marzyeh Ghassemi, Alan Moses
                <br>

                <venue>Machine Learning for Computational Biology (MLCB)</venue>, 2020
                <br>
                <a href="https://www.biorxiv.org/content/10.1101/2020.09.04.283929v2.full.pdf">Paper</a> / <a href="data/CPCProt_MLCB_Poster.pdf">Poster</a> / <a href="https://github.com/amyxlu/CPCProt">Code</a>
                <p>CPCProt uses contrastive learning to learn a parameter-efficient way of embedding proteins, and performs competitively with large language models.</p>
              </td> 
            </tr>

            <!-- paper -->
            <tr>
              <td style="padding:20px;width:25%;vertical-align:middle">
                <img src="images/hurtfulwords.png" alt="SciBERT model trained on medical notes generates biased sentences given the ethnicity of a patient." width="160" height="120" style="border-style: none">
              </td>

              <td width="75%" valign="middle">
                <a href="https://arxiv.org/pdf/2003.11515.pdf" id="hurtfulwords">
                  <papertitle>Hurtful Words: Quantifying Biases in Clinical Contextual Word Embeddings</papertitle>
                </a>
                <br>

                Haoran Zhang*, <strong>Amy X Lu*</strong>, Mohamed Abdalla, Matthew McDermott, Marzyeh Ghassemi. <equalcontribution>*Equal Contribution</equalcontribution><br>

                <venue>ACM Conference on Health, Inference, and Learning (CHIL)</venue>, 2020 &nbsp <font color="red"><strong>(Spotlight)</strong></font><br>
                <a href="https://dl.acm.org/doi/pdf/10.1145/3368555.3384448">Paper</a> / <a href="https://arxiv.org/pdf/2003.11515.pdf">arXiv</a> / <a href="data/HurtfulWordsHintonSymposium.pdf">Poster</a> / <a href="https://github.com/MLforHealth/HurtfulWords">Code</a>

                <p> We apply fairness definitions to quantify the cross-group bias in BERT embeddings pretrained on medical notes, and find statistically significant differences in classifier performance.</p>
              </td> 
            </tr>

            <!-- paper -->
            <tr>
              <td style="padding:20px;width:25%;vertical-align:middle">
                <img src="images/coos.png" alt="CPCProt" width="160" height="120" style="border-style: none">
              </td>

              <td width="75%" valign="middle">
                <a href="https://www.biorxiv.org/content/10.1101/2020.09.04.283929v2.full.pdf" id="CPCProt">
                  <a href="https://proceedings.neurips.cc/paper/2019/file/fc2c7c47b918d0c2d792a719dfb602ef-Paper.pdf" id="coos">
                    <papertitle>The Cells Out of Sample (COOS) dataset and benchmarks for measuring out-of-sample generalization of image classifiers</papertitle>
                  </a>
                  <br>

                  Alex X Lu, <strong>Amy X Lu</strong>, Wiebke Schormann, Marzyeh Ghassemi, David Andrews, Alan Moses.
                  <br>

                  <venue>Neural Information Processing Systems (NeurIPS)</venue>, 2019
                  <br>

                  <a href="https://proceedings.neurips.cc/paper/2019/file/fc2c7c47b918d0c2d792a719dfb602ef-Paper.pdf">Paper</a> / <a href="https://arxiv.org/pdf/1906.07282.pdf">arXiv</a>
                  <p>Introduces the COOS-7 dataset to benchmark and evaluate the capacity of feature learning methods to generalize to natural distribution shifts in microscopy images.</p>
                </td> 
              </tr>

              <!-- paper -->
              <tr>
                <td style="padding:20px;width:25%;vertical-align:middle">
                  <img src="images/network.jpg" alt="Coauthorship network" width="160" height="120" style="border-style: none">
                </td>

                <td width="75%" valign="middle">
                  <a href="https://bmjopen.bmj.com/content/8/6/e019110.full" id="bmj">
                    <papertitle>History and publication trends in the diffusion and early uptake of indirect comparison meta-analytic methods to study drugs: animated coauthorship networks over time</papertitle>
                  </a>
                  <br>

                  Joann K Ban, Mina Tadrous, <strong>Amy X Lu</strong>, Erin A Cicinelli, Suzanne M Cadarette
                  <br>

                  <venue>BMJ open</venue>, 2018
                  <br>
                  <a href="https://bmjopen.bmj.com/content/8/6/e019110.full">Paper</a>
                </td> 
              </tr>
            </p>
          </td>
        </tr>
      </tbody></table>


      <!-- Workshops -->

      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
          <td style="padding:20px;width:100%;vertical-align:middle">
            <p><em><big>Workshop Papers<big></em></p>
          </td>
        </tr>
      </tbody></table>

      
      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>

              <!-- paper -->
              <tr>
                <td style="padding:20px;width:25%;vertical-align:middle">
                  <width="160" height="120" style="border-style: none">
                </td>

                <td width="75%" valign="middle">
                  <a href="https://openreview.net/pdf?id=Dc5J-bcEGW5" id="mldd"><papertitle>Data-Driven Optimization for Protein Design: Workflows, Algorithms and Metrics</papertitle>
                  </a>
                  <br>
                  Sathvik Kolli, <strong>Amy X. Lu</strong>, Xinyang Geng, Aviral Kumar, Sergey Levine
                  <br>

                  <venue>ICLR Workshop on Machine Learning for Drug Discovery (MLDD)</venue>, 2022
                  <br>
                  <a href="https://openreview.net/pdf?id=Dc5J-bcEGW5">Paper</a>
                  <p>Strategies for data curation, model-training, optimization, and evaluation heuristics for data-driven proposals of novel <em>de novo</em> proteins.</p>
                </td>
              </tr>

              <!-- paper -->
              <tr>
                <td style="padding:20px;width:25%;vertical-align:middle">
                  <img src="images/paired-cell-inpainting.png" alt="Paired Cell Inpainting Schematic" width="160" height="120" style="border-style: none">
                </td>

                <td width="75%" valign="middle">
                  <a href="https://drive.google.com/file/d/0B4M2lUVyJzS4d3B2X3AtVmxFTm5iT0dmN2RrVmhCRFY4MGdj/view?resourcekey=0-cZHEJfuqWRhzH3uZSVJGNA" id="icml2019"</a>
                    <papertitle>Paired Cell Inpainting: A Multiple-Instance Extension of Self-Supervised Learning for Bioimage Analysis</papertitle>
                  </a>
                  <br>

                  Alex X. Lu, <strong>Amy X Lu</strong>, Alan Moses
                  <br>

                  <venue>ICML Workshop on Self-Supervised Learning</venue>, 2019
                  <br>
                  <a href="https://drive.google.com/file/d/0B4M2lUVyJzS4d3B2X3AtVmxFTm5iT0dmN2RrVmhCRFY4MGdj/view?resourcekey=0-cZHEJfuqWRhzH3uZSVJGNA">Paper</a>
                </td> 
              </tr>



              <!-- paper -->
              <tr>
                <td style="padding:20px;width:25%;vertical-align:middle">
                  <width="160" height="120" style="border-style: none">
                </td>

                <td width="75%" valign="middle">
                  <a href="https://www.fairmlforhealth.com/accepted-papers" id="CPCProt">
                    <papertitle>Quantifying Fairness in a Multi-Group Setting and its Impact in the Clinical Setting</papertitle>
                  </a>
                  <br>

                  Mohamed Abdalla, <strong>Amy X Lu</strong>, Haoran Zhang, Irene Chen, Marzyeh Ghassemi
                  <br>

                  <venue>NeurIPS Workshop on Fair ML for Health</venue>, 2019
                  <br>
                  <a href="https://www.fairmlforhealth.com/accepted-papers">Paper</a>
                  <p></p>
                </td> 
              </tr>


              <!-- paper -->
              <tr>
                <td style="padding:20px;width:25%;vertical-align:middle">
                  <alt="CPCProt" width="160" height="120" style="border-style: none">
                </td>

                <td width="75%" valign="middle">
                  <a href="https://mlcb.github.io/mlcb2019_proceedings/papers/paper_41.pdf" id='mlcb2019'>
                    <papertitle>Transfer Learning vs. Batch Effects: what can we expect from neural networks in computational biology</papertitle>
                  </a>
                  <br>

                  Alan Moses, Alex X Lu, <strong>Amy X Lu</strong>, Marzyeh Ghassemi
                  <br>

                  <venue>Machine Learning for Computational Biology (MLCB)</venue>, 2019
                  <br>
                  <a href="https://mlcb.github.io/mlcb2019_proceedings/papers/paper_41.pdf">Paper</a>
                </td> 
              </tr>
            </p>
          </td>
        </tr>
      </tbody></table>




      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
          <td style="padding:20px">
            <br>
            <p style="text-align:left">
              <strong>Misc.</strong> I enjoy road biking through the East Bay redwoods, and playing the piano, especially Chopin and hip-hop covers. I'm usually coding to EDM or Beethoven's complete piano sonatas while eating 90% dark chocolate. My car and bikes are named after F. Scott Fitzgerald characters, and administrative entities call me Xiaoping Lu (逯晓萍).
            </p>
          </td>
        </tr>
      </tbody></table>




      <!-- Center aligned credits -->

      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
          <td style="padding:0px">
            <br>
            <p style="text-align:right;font-size:small;">
              Template from <a href="https://jonbarron.info/">here</a>!
            </p>
          </td>
        </tr>
      </tbody></table>
      

    </td>
  </tr>
</table>
</body>
</html>
